<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>professors</class>
 <widget class="QDialog" name="professors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="back">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
	background: #39AAFF;
	color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/back.png</normaloff>:/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>0</y>
     <width>1311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Century Gothic&quot;;
color: rgb(56, 56, 191);
background-color: rgb(199, 197, 199);</string>
   </property>
   <property name="text">
    <string>Professors management</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Professor name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deleteLine">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>680</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1621</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Professors Details</string>
    </attribute>
    <widget class="QTableView" name="professors_table">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>1171</width>
       <height>361</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1211</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Search for professor that you want</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_btn_2">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLine">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Update Professor</string>
    </attribute>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>410</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>237</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Department</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="isChef_2">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>330</y>
       <width>507</width>
       <height>42</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Chef</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneLine_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>260</y>
       <width>451</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLine_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>120</y>
       <width>461</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="departmentCombo_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>330</y>
       <width>451</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLine_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>190</y>
       <width>261</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="prefix_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>180</y>
       <width>235</width>
       <height>54</height>
      </rect>
     </property>
     <property name="text">
      <string>@uae.ac.ma</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idLine_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>461</width>
       <height>44</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Professor Statistics</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Number of Professors</string>
     </property>
    </widget>
    <widget class="QLabel" name="prof_counter">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>210</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Century Gothic&quot;;
color: rgb(6, 0, 203);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>210</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string> Number of Subjects</string>
     </property>
    </widget>
    <widget class="QLabel" name="sub_counter">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>210</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Century Gothic&quot;;
color: rgb(6, 0, 203);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Add Professor</string>
    </attribute>
    <widget class="QPushButton" name="add_btn_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>390</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>237</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Department</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>107</y>
       <width>441</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="nameLine"/>
      </item>
      <item>
       <widget class="QLineEdit" name="emailLine"/>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneLine"/>
      </item>
      <item>
       <widget class="QComboBox" name="departmentCombo"/>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="isChef">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>310</y>
       <width>517</width>
       <height>42</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Chef</string>
     </property>
    </widget>
    <widget class="QLabel" name="prefix">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>150</y>
       <width>255</width>
       <height>104</height>
      </rect>
     </property>
     <property name="text">
      <string>@uae.ac.ma</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="update_btn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>680</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>680</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>delete_btn</zorder>
  <zorder>deleteLine</zorder>
  <zorder>back</zorder>
  <zorder>update_btn</zorder>
  <zorder>tabWidget</zorder>
  <zorder>label_22</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
